teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'unknown')
+    @specialization = specialization
+    super(age, name)
+  end
```

Comment:
- [ ] You are missing the `parent_permission` parameter. Kindly add it and also do the same in the `super` method.